src/main/java/org/mdbenefits/app/inputs/MdBenefitsFlow.java

```diff
@@ -204,6 +204,27 @@ public class MdBenefitsFlow extends FlowInputs {
     @NotEmpty(message = "{error.missing-general}")
     private List<String> additionalIncome;
 
+    @NotEmpty(message = "{error.missing-general}")
```

Comment:
This is clever.
1. The first time that the field appears it will be in the additional income page and the value is set to the name. If I don't select the checkbox then the input will not be submitted so this error message will not be triggered.

2. The second time, only fields that were selected in the previous page will have corresponding inputs so if I try to submit a value on the second page, the value will be empty so it will not let me move forward.